### DISCUSSION TITLE
👻 Not your homie? Or your homie afterall?

### DISCUSSION THREAD

**Discussion by frenck** [frenck/spook#612] [Polls]
Spook will turn 1-year old on [23 February 2024](https://github.com/frenck/spook/commit/67803faf19bca7c8543e0865e1dba58755315652).

I originally started out to create something to experiment with (and I still do experiment concepts with Spook). However, nowadays, Spook seems to be gaining popularity and slowly getting an actual "fan base" (sounds weird to write/say).

The slogan _"Not your Homie"_ I added because, well, it is a custom integration; custom integrations should be avoided as much as possible. Additionally, there is stuff in Spook that might not be the best idea.

Spook 👻 is turning one year old. How it developed and was embraced by part of the community, made me wonder: Should I change its narrative?

---

Related poll: [Change the license of Spook?](https://github.com/frenck/spook/discussions/613)

**Comment by Mariusthvdb** [comment]
seems I am the only one.... but using Spook as power tool, so not using it (as we are not allowed (maybe change that too)) is not an option....

Makes me say, yes, change it, and why not to the functionality it offers: Power Tools.

(remember the days when we had a set of power tools  for windows, which also made life easier in the situations where regular OS left us wanting.)

For Spook it is just the same: HA OS is fine, but there are situations we need, well, a Power tool.

btw Congrats!

**Comment by boheme61** [comment]
> but using Spook as power tool, so not using it (as we are not allowed (maybe change that too)) is not an option

I actually said "keep it" for the same reasons( Ironically ), as i find it slightly "annoying", when someone repeatedly reminding me about things i have to fix, or things i done wrong/bad ( it's like a nagging B.... , or a besserwisser  ) But in the end we like won't live without them, because they give us so much more ( And they are right !, from their perspective ... maybe )

So if it's have to be change, i would like to see ( Spook , maybe Your Best Homie after-all )

PS: Above Slogan is copyright© protected in 8 continents !

**Comment by jsnfwlr** [comment]
"Spook: Not for the faint of heart" may be a better slogan

**Comment by WildRat** [comment]
The current name is the reason I haven't known or used this great tool for almost a year since its release. I find it a bit confusing for newcomers and think it's preventing its use from becoming widespread. In my opinion, it’s better to simply “Spook” or, as previously suggested, “Power Tool”.

**Comment by frenck** [comment]
![logo_wordmark_catchphrase_2048x2048](https://github.com/frenck/spook/assets/195327/80c53996-1a1c-4ee6-9342-96c3667a6bd8)

**Reply by frenck** [reply]
There it goes... thanks for the votes and comments.

Spook will become our friendly ghost :)

**Comment by JensJackowski** [comment]
Using costum integrations and take same risks can have many advantages. 

Usually i buy things and learn after the purchages how to use them. And very often i have a learning-curve that provides me with knowlegde that i should have had when i decide what to buy.

This means i face often the problem that i want to get things work just because i already have them because i care about the enviroment. Smarthomestuff is usually so cheap that it is not realy an economic question to buy the "proper thing" after i have understood what thing is right. But thinking about all the effeort and ressources that are used to produce the "wrong thing" makes me to try to get i work.

Unfortnuatly is "try and error" often the only methode to get my things work. Normaly the lack of documentation is the main reason why thinks do not interact with HomeAssistent. And than it is easy to forget or even see the overload to my homeassistant configuration with i am generating with all my attemds.

Spook help me and make it much more easy to clean up. Naturally it is generally stupid to clean up dirt when you can work in a way that avoids dirt. 

But when trying and testing and learning this way, it is okay to make sameting dirty.

And i am very happy about you and spook. You provide a vaccum cleaner for easy cleaning istead of shouting and questing the way i produced the dirt :)
